AIPscan/Aggregator/views.py

```diff
@@ -206,8 +206,12 @@ def new_fetch_job(fetch_job_id):
     datetime_obj_start = datetime.now().replace(microsecond=0)
     timestamp_str = datetime_obj_start.strftime("%Y-%m-%d-%H-%M-%S")
     timestamp = datetime_obj_start.strftime("%Y-%m-%d %H:%M:%S")
-    os.makedirs("AIPscan/Aggregator/downloads/" + timestamp_str + "/packages/")
-    os.makedirs("AIPscan/Aggregator/downloads/" + timestamp_str + "/mets/")
+
+    try:
+        os.makedirs(f"AIPscan/Aggregator/downloads/{timestamp_str}/packages/")
+        os.makedirs(f"AIPscan/Aggregator/downloads/{timestamp_str}/mets/")
+    except PermissionError:
+        return jsonify({"message": "Download directory misconfigured"}), 500
```

Comment:
I'd put "permissions error" or something similar in the error message to help the operator pinpoint the problem.

Reply:
Good call!